src/main/kotlin/com/devfloor/untitled/articleoption/application/ArticleOptionService.kt

```diff
@@ -13,4 +14,16 @@ class ArticleOptionService(
     @Transactional(readOnly = true)
     fun showAllByArticle(article: Article): List<ArticleOption> =
         repository.findAllByArticle(article)
+
+    @Transactional
+    fun createAllByOptions(article: Article, options: List<Option>) {
+        options.forEach {
+            repository.save(ArticleOption(article, it))
+        }
+    }
+
+    @Transactional
+    fun createByOption(article: Article, option: Option) {
+        repository.save(ArticleOption(article, option))
+    }
```

Comment:
여기도 앞에서 처럼 Transactional이 불필요해보입니다!
```suggestion
    fun createByOption(article: Article, option: Option) {
        repository.save(ArticleOption(article, option))
    }
```